kubeflow/env.sh

```diff
@@ -1,12 +1,39 @@
-export KF_NAME=<kubeflow-cluster-name>
+# 1. Edit <placeholders>.
+# 2. Other env vars are configurable, but with default values set below.
+
+# Your Google Cloud project ID
 export KF_PROJECT=<gcp-project-id>
-export KF_PROJECT_NUMBER=$(gcloud projects describe ${KF_PROJECT} --format='value(projectNumber)')
-export KF_DIR=<kubeflow-download-path>
-export MGMT_NAME=<management-cluster-name>
-export MGMTCTXT=${MGMT_NAME}
-export LOCATION=<zone>
+# You can get your project number by running this command:
+# gcloud projects describe --format='value(projectNumber)' "${KF_PROJECT}"
+export KF_PROJECT_NUMBER=<gcp-project-number>
+# Kubeflow Admin email address
+# Example: admin@gmail.com
 export ADMIN_EMAIL=<administrator-full-email-address>
+# The management cluster name you have already deployed following:
+# https://www.kubeflow.org/docs/distributions/gke/deploy/management-setup/
+export MGMT_NAME=<management-cluster-name>
+# Management cluster kubectl context name, defaults to ${MGMT_NAME} if you
+# followed the above guide.
+export MGMTCTXT="${MGMT_NAME}"
 
-export ASM_LABEL=asm-192-1
+# The following env vars are also configurable.
 
+# KF_NAME is your Kubeflow cluster name.
+# It should satisfy the following conditions:
+# * is unique within your project.
+# * alphanumeric characters and hyphen "-" only.
+# * starts with a letter.
+# * ends with an alphanumeric character.
+export KF_NAME=kubeflow
```

Comment:
Should we pre-set this value for user? This variable is very frequently used by user to identify and maintain a cluster, so I think user should explicitly set them: KF_NAME=<Kubeflow-cluster-name>

Reply:
I meant, they can always change it as documented above. So provide a valid default value sounds better than an invalid one, what do you think?

Reply:
It sounds good for providing a starting point for new users.